modality-probe-graph/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "modality-probe-graph"
+version = "0.1.0"
+authors = ["dan pittman <dan@auxon.io>"]
+edition = "2018"
+
+[features]
+test_support = ["chrono"]
```

Comment:
I recommend making the default features explicit by including an entry for it with an empty array, i.e. `default = []`